test/input.txt

```diff
@@ -148,6 +148,24 @@
   clear
   list
 
+##########################################################
+# test sort command
+##########################################################
+    # insert two person
+    add Bob p/1234567 e/bob@gmail.com a/abc
+    add Alice p/1234567 e/Alice@gmail.com a/abc
+    list
+    sort
```

Comment:
It might be more appropriate to swap sort and list, so that you can make use of the `list` to check if your sorting was done correctly.